pygenometracks/tracks/HiCMatrixTrack.py

```diff
@@ -89,26 +89,26 @@ def set_properties_defaults(self):
         super(HiCMatrixTrack, self).set_properties_defaults()
         region = None
         if self.properties['region'] is not None:
+            chrom = self.properties['region'][0]
+            # The chromosome name will be UCSC format because
+            # cooler v0.8.5 can fetch UCSC format in Ensembl-like format
+            # but not the contrary
+            if not chrom.startswith('chr'):
```

Comment:
what if it is mitochondrial dna or some contigs?

Reply:
I agree it would be better to be able to catch the exception from HiCMatrix but for the moment we cannot as it is `exit`.

Reply:
I will try to see how it behave.

Reply:
it is a very bad practice to add chr to just everything, that is why cooler doesn't do that too, i suppose.  We already have tests to check for ensemble vs ucsc and i think that failure message in hicmatrix makes sense. This is my opinion. Maybe others have better idea.

Reply:
What do you mean by 

>  We already have tests to check for ensemble vs ucsc